core/src/main/scala/io/snappydata/util/ServiceUtils.scala

```diff
@@ -97,8 +97,8 @@ object ServiceUtils {
 
     "jdbc:" + Constant.SNAPPY_URL_PREFIX + (if (locatorUrl.contains(",")) {
       locatorUrl.substring(0, locatorUrl.indexOf(",")) +
-          ";secondary-locators=" + locatorUrl.substring(locatorUrl.indexOf(",") + 1)
-    } else locatorUrl)
+          "/;secondary-locators=" + locatorUrl.substring(locatorUrl.indexOf(",") + 1 ) + "/"
```

Comment:
The issue with multiple locator is fixed. is there any other issue with multiple URL's that I am not aware of?